.github/workflows/test_tutorial_workflow.yml

```diff
@@ -19,13 +19,16 @@ concurrency:
 
 jobs:
   test:
+    strategy:
+      matrix:
+        python-version: ['3.7', '3.11']
```

Comment:
Consider using `3.x` to avoid having to update this number.

```suggestion
        python-version: ['3.7', '3.x']
```

Reply:
I changed this for a reason, but I think that reason related to an unrelated test failure.

Reply:
That error is coming from here https://github.com/cylc/cylc-doc/blob/151dc587f1d338f6a1c3730fb51100948168c7a4/.github/actions/install-dependencies/action.yml#L31

What happens if you just specify `3` instead of `3.x`?

Reply:
That's the current state of the PR
QED

Reply:
Hmm, conda is installing Python 3.10: https://github.com/cylc/cylc-flow/actions/runs/4881596596/jobs/8710692879?pr=5513#step:4:491

I'm not sure which version is being used by the tests

Reply:
@wxtim Click the link I sent, it's the same as yours but with a link to the line number confirming conda is installing python 3.10

Reply:
Ah, didn't realise we had another environment, I think that's the version of Python which is used to run the tutorial scripts?

Conda/Mamba uses a flexible solver so may pick a lower version of Python to allow other dependencies to be bumped.

So if we definitely want 3.11 we will have to explicitly state it, soz.